lib/iris/plot.py

```diff
@@ -662,7 +663,18 @@ def _ensure_cartopy_axes_and_determine_kwargs(x_coord, y_coord, kwargs):
     # Ensure the current axes are a cartopy.mpl.GeoAxes instance.
     axes = kwargs.get('axes')
     if axes is None:
-        _replace_axes_with_cartopy_axes(cartopy_proj)
+        if (isinstance(cs, iris.coord_systems.RotatedGeogCS) and
```

Comment:
it is plausible that we could include GeogCS in this behaviour control.  It is less of an issue in our user community as the use of GeogCS for limited area models is unusual

in my view this is a corner case, it is not a generalised problem.  There are limitations in Cartopy's interfaces and usage patterns with particular data sets which conflict in this small set of cases, so I am looking for a limited scope fix, not a generalised solution

@cpelley Please may you clarify: are you explicitly asking for this behaviour to be provided for and tested for `GeogCS` as part of this change?